### ISSUE TITLE
is: [] not found in the *.raml file

### ISSUE THREAD

**Issue by przezor** [mulesoft/raml-object-to-raml#2]
Object:

```
resources: 
   [ { is: [Object],
       uriParameters: [Object],
       relativeUri: '/test/',
       methods: [Object],
       ...
```

In this case `is: [blabla]` will not appear in the *.raml file. Would you be so kind to fix it?

Cheers,

Luke

**Comment by przezor** [comment]
I found out that adding this code solves the problem:

```
  if (resource.is) {
      child.is = resource.is;
    }
```

**Comment by blakeembrey** [comment]
Released with `0.0.6`.
